panoramapublic/src/org/labkey/panoramapublic/pipeline/CopyExperimentFinalTask.java

```diff
@@ -650,11 +650,11 @@ private User createReviewerAccount(String reviewerEmailPrefix, String password,
         return newUser.getUser();
     }
 
-    private void assignReader(UserPrincipal reader, Container target)
+    private void assignReader(UserPrincipal reader, Container target, User pipelineJobUser)
     {
         MutableSecurityPolicy newPolicy = new MutableSecurityPolicy(target, target.getPolicy());
         newPolicy.addRoleAssignment(reader, ReaderRole.class);
-        SecurityPolicyManager.savePolicy(newPolicy);
+        SecurityPolicyManager.savePolicy(newPolicy, pipelineJobUser);
```

Comment:
Same here. It should be OK to pass in the User that started the pipeline job instead of User.getAdminServiceUser().